src/routes/profile/index.tsx

```diff
@@ -49,7 +49,7 @@ export default class Profile extends Component<Props, State> {
 				<div>Current time: {new Date(time).toLocaleString()}</div>
 
 				<p>
-					<Button raised primary ripple onClick={this.increment}>Click Me</Button>
+					<Button raised onClick={this.increment}>Click Me</Button>
```

Comment:
`primary` seems to be unsupported and does nothing, but `ripple` is a valid prop